.github/DISCUSSION_TEMPLATE/resources.yml

```diff
@@ -0,0 +1,49 @@
+title: "<resource title>"
+labels: ["Resource"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        ## Resource Submission
+        
+        Please provide the following details about the resource you are sharing. 
+        
+        Note that duplicate resources may be removed to keep things tidy.
+  - type: textarea
+    id: context
+    attributes:
+      label: Context
+      description: 'Provide any additional context about the resource. Why is it useful? Who might it benefit?'
+      placeholder: 'This guide is useful for those looking to understand React from scratch.'
+    validations:
+      required: true
+  - type: input
+    id: url
+    attributes:
+      label: URL
+      description: Provide the most direct link to the resource
+      placeholder: 'https://example.com'
+    validations:
+      required: true
+  - type: textarea
+    id: extra
+    attributes:
+      label: Additional information
+      description: "Share any additional links or information here."
+      value: |
+        ...
+    validations:
+      required: false
+  - type: checkboxes
+    id: labels
+    attributes:
+      label: 'Labels'
+      description: 'Select appropriate labels for this resource (you can select multiple).'
+      options:
+        - label: 'documentation'
+        - label: 'tools'
+        - label: 'best-practices'
+        - label: 'tutorial'
+        - label: 'other'
+    validations:
+      required: true
```

Comment:
I like the idea of having some validation of the labels but on the other hand, it seems very constraining. A huge benefit I would see in this resource library would be the ability to find all resource related to, e.g., 'Testing', or 'Continuous Integration', etc.
The ability for submitters and maintainers to add arbitrary tags seems like a necessary part of this. :thinking:

Reply:
I agree labels are incredibly helpful. The motivation to prespecify was the experience with the tags for the blogs - which became repetitive, redundant, and cluttering (e.g., `S3` and `S3 Class` categories both with only one entry).

More flexibility is not always better, but if we want we could leave the suggested labels open for user entry. The curation then happens by adding labels to the discussion directly (so the form ≠ discussion labels immediately) so we don't create the repetition and redundancy because of slight differences in formulation.

Reply:
> The motivation to prespecify was the experience with the tags for the blogs - which became repetitive, redundant, and cluttering (e.g., S3 and S3 Class categories both with only one entry).

Thanks for flagging, I have made an attempt at harmonizing them in https://github.com/epiverse-trace/epiverse-trace.github.io/pull/240

> More flexibility is not always better,

No disagreement from me here :+1:, it's always a trade-off

> The curation then happens by adding labels to the discussion directly (so the form ≠ discussion labels immediately) so we don't create the repetition and redundancy because of slight differences in formulation.

I much prefer this solution and would encourage it. Feel free to take the final decision for this iteration and we can revisit later down the line once we start having a couple dozen resources.